build.xml

```diff
@@ -1665,7 +1763,7 @@
     <echo message="Mem size : ${mem.size}"/>
   </target>
 
-  <target name="test" depends="build-test,get-cores,get-mem,stress-build" description="Parallel Test Runner">
+  <target name="test" depends="build-test" description="Parallel Test Runner">
```

Comment:
Why does this not depend on `get-cores,get-mem` anymore? Otherwise, those targets are unused in the build.xml

Reply:
Oops, re-added